Makefile

```
@@ -73,26 +74,35 @@ ifeq ($(MAKECMDGOALS),statusgo-ios-library)
     CGO_LDFLAGS+=-Os -flto
 endif
 
+CGO_CFLAGS = -I/$(JAVA_HOME)/include -I/$(JAVA_HOME)/include/darwin
+export GOPATH ?= $(HOME)/go
+
+GIT_ROOT ?= $(dir $(realpath $(lastword $(MAKEFILE_LIST))))
+GIT_COMMIT ?= $(shell git rev-parse --short HEAD)
+GIT_AUTHOR ?= $(shell git config user.email || echo $$USER)
+
+export NIM_SDS_HEADER_PATH=$(GIT_ROOT)/vendor/github.com/waku-org/sds-go-bindings/third_party/nim-sds/library
+export NIM_SDS_LIB_PATH=$(GIT_ROOT)/vendor/github.com/waku-org/sds-go-bindings/third_party/nim-sds/build
+export CGO_CFLAGS=-I${NIM_SDS_HEADER_PATH}/
+export CGO_LDFLAGS=-L${NIM_SDS_LIB_PATH}/ -lsds -Wl,-rpath,${NIM_SDS_LIB_PATH}/
```

Use spacing consistent with rest of the `Makefile`:
```suggestion
export NIM_SDS_HEADER_PATH = $(GIT_ROOT)/vendor/github.com/waku-org/sds-go-bindings/third_party/nim-sds/library
export NIM_SDS_LIB_PATH = $(GIT_ROOT)/vendor/github.com/waku-org/sds-go-bindings/third_party/nim-sds/build
export CGO_CFLAGS = -I${NIM_SDS_HEADER_PATH}/
export CGO_LDFLAGS = -L${NIM_SDS_LIB_PATH}/ -lsds -Wl,-rpath,${NIM_SDS_LIB_PATH}/
```